thirdparty/cmdconfig/commands/cmdeval/cmdeval_test.go

```diff
@@ -41,7 +41,7 @@ metadata:
   name: function-input
   annotations:
     config.kubernetes.io/function: |
-      container: {image: 'foo:bar'}
+      container: {image: 'gcr.io/kpt-fn/foo:bar'}
```

Comment:
@droot @Shell32-Natsu Why is this annotation used in next branch?

Reply:
@Shell32-Natsu This shouldn't be needed anymore right ? I remember this was required initially due to the technical debt we have accumulated as part of `eval` because `eval` still uses the legacy `kpt run` code, not sure if it is still true.

Reply:
Modified logic. This PR will not make any changes to this file.